bin/generate_unfiltered_sce.R

```diff
@@ -152,6 +157,19 @@ if (opt$feature_dir != "") {
   unfiltered_sce <- merge_altexp(unfiltered_sce, feature_sce, opt$feature_name)
   # add alt experiment features stats
   altExp(unfiltered_sce, opt$feature_name) <- scuttle::addPerFeatureQCMetrics(altExp(unfiltered_sce, opt$feature_name))
+
+  # read in cellhash pool if exists
+  if (!is.null(opt$cellhash_pool_file)) {
+    if (!file.exists(opt$cellhash_pool_file)) {
+      stop("Can't find cellhash_pool_file")
+    }
```

Comment:
NB Josh recently discovered that `stopifnot()` got better when we weren't looking! I am not suggesting to change this code, just sharing the good news - https://blog.r-hub.io/2022/03/10/input-checking/

Reply:
Just to make this more explicit, now we can do things like: 
```suggestion
    stopifnot("Can't find cellhash_pool_file" = file.exists(opt$cellhash_pool_file))
```